student/models.py

```diff
@@ -24,6 +24,8 @@ class Student(models.Model):
     phone_number = models.CharField(max_length=15, blank=True)
     educational_institution = models.ForeignKey(EducationalInstitution, on_delete=models.PROTECT, default=None,
                                                 related_name="students")
+    major = models.CharField(max_length=255)
+    about = models.TextField(blank=True)
```

Comment:
what is this field?

Reply:
The major field is to specify which major the student is pursuing (which I will change to a choice field).
The about field is a free text field where the student can write anything about themselves, from work experience to projects they've done.